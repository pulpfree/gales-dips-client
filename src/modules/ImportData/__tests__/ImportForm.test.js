/* eslint no-undef: "off" */

import React from 'react'
import { shallow } from 'enzyme'

import ImportForm from '../ImportForm'

describe('ImportForm', () => {
  let mountedC
  let props
  const reportC = () => {
    if (!mountedC) {
      mountedC = shallow(<ImportForm {...props} />)
    }
    return mountedC
  }

  beforeEach(() => {
    props = {
      history: { location: { pathname: '/' } },
    }
    mountedC = undefined
  })

  it('matches snapshot', () => {
    expect(reportC()).toMatchSnapshot()
  })
})
