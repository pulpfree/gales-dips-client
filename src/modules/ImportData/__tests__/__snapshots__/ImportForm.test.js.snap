// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImportForm matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(ImportForm)
    history={
        Object {
            "location": Object {
              "pathname": "/",
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "classes": Object {
        "container": "ImportForm-container-1",
        "formCell": "ImportForm-formCell-4",
        "formContainer": "ImportForm-formContainer-2",
        "formControl": "ImportForm-formControl-5",
        "formRow": "ImportForm-formRow-3",
        "processMsg": "ImportForm-processMsg-8",
        "radioGroup": "ImportForm-radioGroup-6",
        "rightIcon": "ImportForm-rightIcon-7",
        "submitButton": "ImportForm-submitButton-9",
      },
      "history": Object {
        "location": Object {
          "pathname": "/",
        },
      },
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "classes": Object {
          "container": "ImportForm-container-1",
          "formCell": "ImportForm-formCell-4",
          "formContainer": "ImportForm-formContainer-2",
          "formControl": "ImportForm-formControl-5",
          "formRow": "ImportForm-formRow-3",
          "processMsg": "ImportForm-processMsg-8",
          "radioGroup": "ImportForm-radioGroup-6",
          "rightIcon": "ImportForm-rightIcon-7",
          "submitButton": "ImportForm-submitButton-9",
        },
        "history": Object {
          "location": Object {
            "pathname": "/",
          },
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
